public/js/backup.js

```diff
@@ -5,7 +5,7 @@ const Backup = {};
 
 Backup.initializeAll = function () {
     // bind events to DOM
-    $(document).on("click.return", "#return", () => { window.location.replace("./"); });
+    $(document).on("click.return", "#return", () => { window.location.href = "/"; });
```

Comment:
You can revert those back if you want to, I have no strong opinion on one version or the other 👍 Sorry for the confusion!